samples/sample_hevc_fei/src/sample_hevc_fei.cpp

```diff
@@ -424,6 +424,7 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU32 nArgNum, sInputParams&
         }
         else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-ForceCtuSplit")))
         {
+            params.encodeCtrl.ForceCtuSplit = 1; // for display order mode (w/o preenc)
```

Comment:
what is wrong with encoded order (I think nothing)? And what will happen in case of PreENC + ENCODE?
I don't see any additional checks in this file

Reply:
@mgonchar, removed a confusing comment and added checks.